base/src/main/scala/co/uproot/abandon/Config.scala

```diff
@@ -282,6 +283,20 @@ object SettingsHelper {
     val alias = config.optional("alias") { _.getString(_) }
     AccountSettings(ASTHelper.parseAccountName(name), alias)
   }
+
+  def ensureInputProtection(inputs: Set[String], settings: Settings): Unit = {
+    def filenamesToPaths: Seq[String] => Seq[Path] = _.map(Paths.get(_).normalize)
+    def filenamesToConfRelPaths: Seq[String] => Seq[String] = _.filterNot(_ equals "-").map(settings.getConfigRelativePath)
+
+    val inputFiles = filenamesToPaths(inputs.toSeq)
+    val repOutFiles = filenamesToPaths(settings.reports.flatMap(rep => filenamesToConfRelPaths(rep.outFiles)))
+    val expOutFiles = filenamesToPaths(settings.exports.flatMap(exp => filenamesToConfRelPaths(exp.outFiles)))
```

Comment:
You're right, it makes sense to make have that part in the Settings class. 
It seems to me that this part (App.scala, line 10) could be moved to the Settings class, too:

`val writesToScreen = outFiles.contains("-") || outFiles.isEmpty`

What do you think?